Firmware/RTK_Surveyor/Begin.ino

```diff
@@ -265,6 +285,8 @@ void beginSD()
 
   online.microSD = false;
   gotSemaphore = false;
+
+  log_d("settings.enableSD: %d", settings.enableSD);
```

Comment:
I don't mind log_ds, but do you still need this?

Reply:
Ah, no. This can go. I was bug-chasing...